flink-yarn/src/main/java/org/apache/flink/yarn/YarnClusterDescriptor.java

```diff
@@ -1441,13 +1444,17 @@ public String getClusterDescription() {
                 totalMemory += res.getMemory();
                 totalCores += res.getVirtualCores();
                 ps.format(format, "NodeID", rep.getNodeId());
-                ps.format(format, "Memory", res.getMemory() + " MB");
+                ps.format(format, "Memory", getDisplayMemory(res.getMemory()));
```

Comment:
I prefer using the `MemorySize  of flink`  instead of `StringUtils.byteDesc  of hadoop`, because flink already has similar feature. WDYT?

```suggestion
                ps.format(format, "Memory", MemorySize.ofMebiBytes(128).toHumanReadableString()));
```

And I have a demo here.

![image](https://user-images.githubusercontent.com/38427477/222614772-35e3c26d-3406-4172-ade6-5a62a465c654.png)

Reply:
I will modify the code.